content/2-4.md

```diff
@@ -61,54 +61,59 @@ Skills required for this module.
+* explain how table markup supports adaptive strategies, such as displaying tables using custom style sheets and on different screen sizes
+* identify tables that present data and mark up such data tables using the HTML element `table`
+* identify and mark up table headers using the HTML element `th`
+* identify and mark up table data cells using the HTML element `td`
+* identify tables that are used for layout purposes rather than to present data, and code such layout using CSS techniques instead of using HTML structures
 
 #### Teaching Ideas for Topic
 
 Optional ideas to teach the learning outcomes:
 
 * Define simple tables as those which contain one row or column header. Explain that speech and mainstream technologies can associate information in the header and data cells if they are marked up appropriately. Examples of simple tables are provided in the WAI tutorials on [Tables with one header](https://www.w3.org/WAI/tutorials/tables/one-header/).
 * Show examples of use of the HTML elements `table`, `th`, and `td`. Explain that `table` is used to mark up the table layer, `th` is used to code header cells, and `td` is used to code data cells. For examples of how to mark up a simple table, refer to technique [H51: Using table markup to present tabular data](https://www.w3.org/WAI/WCAG21/Techniques/html/H51).
-* Demonstrate use of voice commands, keystrokes, or gestures provided by speech and mainstream technologies to navigate web pages through tables, and to move between cells in a table. Explore advanced functionality that some tools provide, such as presenting all tables in a list where users can select the table they are interested in. Examples of how people with disabilities interact with the web are provided in [Stories of Web Users](https://www.w3.org/WAI/people-use-web/user-stories/).
-* Provide examples of alternative renderings for tables, such as splitting the table into several lists, or showing only parts of the table in a mobile viewport.
+* Demonstrate the use of voice commands, keystrokes, or gestures provided by speech and mainstream technologies to navigate tables, including moving between header and data cells. Explore advanced functionality that some tools provide, such as presenting tables as linear information. Examples of how people with disabilities interact with the web are provided in [Stories of Web Users](https://www.w3.org/WAI/people-use-web/user-stories/).
+* Provide examples of alternative renderings for tables, such as splitting the table into several lists, or showing only parts of the table in a mobile viewport. Explain that layout tables should be avoided, and CSS techniques for visual design should be used instead. Mention that, if a layout table still needs to be used, marking up the table element using `role="presentation"` help assistive technologies present the information contained in the layout table without the table semantics.
```

Comment:
"marking up the table element using `role="presentation"` *will* help assistive technologies present the information". Added in missing 'will'

Reply:
Thanks, will add.